### PR TITLE
migrate_options_shared:Test migrate setmaxdowntime

### PR DESCRIPTION
**Pull request by dzhengfy**
This is to test migrate-setmaxdowntime and migrate-getmaxdowntime during
migration.

Signed-off-by: Dan Zheng <dzheng@redhat.com>

### CODE REVIEW

**Comment by dzhengfy** [comment]
Depends on https://github.com/avocado-framework/avocado-vt/pull/2034

**Comment by dzhengfy** [comment]
(1/1) type_specific.io-github-autotest-libvirt.virsh.migrate_options_shared.positive_test.set_get_maxdowntime_during_migrate.without_postcopy: PASS (152.48 s)
 (1/1) type_specific.io-github-autotest-libvirt.virsh.migrate_options_shared.positive_test.compress_methods.xbzrle_method.without_postcopy: PASS (144.21 s)

File: libvirt/tests/src/migration/migrate_options_shared.py [hunk 1]
```diff
@@ -308,28 +308,69 @@ def check_setspeed(params):
...
 
         :param params: the parameters used
         :raise: test.fail if the value of given item is unexpected
         """
+        def search_jobinfo(jobinfo):
+            """
+            Find value of given item in domjobinfo
+
+            :param jobinfo: cmdResult object
+
```
Comment (kylazhang) [inline libvirt/tests/src/migration/migrate_options_shared.py]: you can remove the blank for uniformity

File: libvirt/tests/src/migration/migrate_options_shared.py [hunk 2]
```diff
@@ -308,28 +308,69 @@ def check_setspeed(params):
             test.fail("Migration speed is expected to be '%d MiB/s', but '%d MiB/s' "
                       "found" % (expected_value, actual_value))
 
-    def check_domjobinfo(params):
+    def check_domjobinfo(params, option=""):
         """
         Check given item in domjobinfo of the guest is as expected
 
         :param params: the parameters used
```
[the remark this answers is not in the document]
Comment (dzhengfy) [reply libvirt/tests/src/migration/migrate_options_shared.py]: updated

File: libvirt/tests/src/migration/migrate_options_shared.py (hunk 1 above)
Comment (dzhengfy) [reply libvirt/tests/src/migration/migrate_options_shared.py]: updated

**Pull request merged**